@here/harp-mapview/lib/geometry/overlayOnElevation.ts

```diff
@@ -31,12 +31,18 @@ function overlayObject(object: TileObject, displacementMap: THREE.DataTexture):
 
     if (Array.isArray(material)) {
         material.forEach(mat => {
-            if (hasDisplacementFeature(mat)) {
+            if (hasDisplacementFeature(mat) && mat.displacementMap !== displacementMap) {
                 mat.displacementMap = displacementMap;
+                mat.needsUpdate = true;
```

Comment:
```suggestion
            if (hasDisplacementFeature(mat) && mat.displacementMap !== displacementMap) {
                mat.displacementMap = displacementMap;
                mat.needsUpdate = true;
                mat.displacementMap.needsUpdate = true;
```

Reply:
Missing setting the needsUpdate flag for the displacement texture

Reply:
I moved this code to DisplacementFeature.ts#setDisplacementMapToMaterial and set needsUpdate flag on the texture as well